firmware/application/apps/ui_recon.cpp

```diff
@@ -717,16 +653,9 @@ ReconView::ReconView(NavigationView& nav)
             input_file = result[0];
             output_file = result[1];
             freq_file_path = get_freqman_path(output_file).string();
-            recon_save_config_to_sd();
 
-            autosave = persistent_memory::recon_autosave_freqs();
-            autostart = persistent_memory::recon_autostart_recon();
-            filedelete = persistent_memory::recon_clear_output();
-            load_freqs = persistent_memory::recon_load_freqs();
-            load_ranges = persistent_memory::recon_load_ranges();
-            load_hamradios = persistent_memory::recon_load_hamradios();
-            update_ranges = persistent_memory::recon_update_ranges_when_recon();
-            auto_record_locked = persistent_memory::recon_auto_record_locked();
+            load_persisted_settings();
```

Comment:
@gullradriel There's a slight change in behavior here that I wanted to double check with you. Before, you weren't re-reading the `continuous` value from pmem. Was that intentional or just missed?

Reply:
It's a miss. Since I'm only disabling loop/continuous on rare times it didn't hurt me before.
Should be added too, for consistency.